test/runAll.unmocked.2.spec.js

```diff
@@ -107,14 +107,21 @@ describe('runAll', () => {
     expect(console.printHistory()).toMatchInlineSnapshot(`
       "
       LOG Preparing... [started]
-      LOG Preparing... [failed]
-      LOG → Merge state could not be restored due to an error!
```

Comment:
Previously the test failed earlier, since the merge status was restored here. Now it more "correctly" fails in the step `Reverting to original state because of errors...`